modules/worker/scripts/worker.template.sh

```diff
@@ -35,11 +36,6 @@ do
   firewall-offline-cmd --zone=$z --add-port=10250/tcp
   firewall-offline-cmd --zone=$z --add-port=10255/tcp
   firewall-offline-cmd --zone=$z --add-port=8472/udp
-
-# node ports
-  firewall-offline-cmd --zone=$z  --add-port=30000-32767/tcp
-  firewall-offline-cmd --zone=$z  --add-port=30000-32767/udp
```

Comment:
We don't have a cloud controller here. We have an oci load balancer which functions as an external load balancer. This talks to an ingress controller (nginx) which in this case is of type NodePort. See here: https://kubernetes.github.io/ingress-nginx/deploy/baremetal/#over-a-nodeport-service and the script: https://github.com/oracle-terraform-modules/terraform-oci-olcne/blob/master/modules/operator/scripts/install_nginx.template.sh

Reply:
Yep I understand we're using an external LB but there's no point to open these firewall ports

Reply:
Does your LB come online then?

Reply:
If you mean #5 then this doesn't change the behavior, I reported that issue using the master branch

Reply:
No, I meant does your LB achieve healthy status?